pkg/controller/account/account_controller.go

```diff
@@ -333,28 +335,25 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 	}
 
 	if (currentAcctInstance.Status.State == "") && (currentAcctInstance.Status.Claimed == false) {
-		// before doing anything make sure we are not over the limit if we are just error
-		accountTotal, err := TotalAwsAccounts(awsSetupClient)
-		if err != nil {
-			reqLogger.Info("Failed to get AWS account total from AWS api", "Error", err.Error())
-			return reconcile.Result{}, err
-		}
-
-		if accountTotal >= awsLimit {
-			reqLogger.Error(ErrAwsAccountLimitExceeded, "AWS Account limit reached", "Account Total", accountTotal)
-			return reconcile.Result{}, ErrAwsAccountLimitExceeded
-		}
 
 		var awsAccountID string
 
 		if currentAcctInstance.Spec.AwsAccountID == "" {
+			// before doing anything make sure we are not over the limit if we are just error
+			accountTotal := totalaccountwatcher.TotalAccountWatcher.Total
+			if accountTotal >= AwsLimit {
```

Comment:
Any reason not to make this
```		
if totalaccountwatcher.TotalAccountWatcher.Total >= AwsLimit {
```

Reply:
No problem. Chose it for style.

Reply:
The value is used in the log message as well.

Reply:
Nope implementing